lms/templates/learner_dashboard/program_details.html

```diff
@@ -13,7 +13,77 @@
 
 <%block name="js_extra">
 <%static:require_module module_name="js/learner_dashboard/program_details_factory" class_name="ProgramDetailsFactory">
-ProgramDetailsFactory({});
+ProgramDetailsFactory({programData:
+    {
+        uuid: '12-ab',
```

Comment:
@schenedx what are you planning to do with this stubbed data? Checking it in, commented out, might be a helpful guide when implementing the backend portion.